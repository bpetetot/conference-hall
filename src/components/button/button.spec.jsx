/* eslint-env jest */
import snap from 'tests/snapshot'
import React from 'react'
import Button from './index.js'

const snapshot = props => snap(Button)({ ...props })

describe('components/button', () => {
  it(
    'should render with children',
    snapshot({
      href: 'someurl',
      children: 'Benjamin',
      className: 'test',
      secondary: true,
    }),
  )
  it(
    'should renter with function children',
    snapshot({
      children: () => <div>some content</div>,
      className: 'test',
      secondary: true,
    }),
  )
})
