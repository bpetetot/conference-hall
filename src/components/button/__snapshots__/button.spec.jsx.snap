// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/button should render with children 1`] = `
<a
  className="cc-button cc-button-secondary test"
  href="someurl"
>
  Benjamin
</a>
`;

exports[`components/button should renter with function children 1`] = `
<div>
  some content
</div>
`;
